tooling/karaf-maven-plugin/src/main/java/org/apache/karaf/tooling/client/DeployMojo.java

```diff
@@ -199,7 +199,7 @@ public String getUpdatedPassword(ClientSession session, String prompt, String la
                 print.println("bundle:install -s " + location);
             }
 
-            final ClientChannel channel = session.createChannel("exec", print.toString().concat(NEW_LINE));
+            final ClientChannel channel = session.createChannel("exec", writer.toString().concat(NEW_LINE));
```

Comment:
Why using the writer whereas we have the printwriter ?

```
            StringWriter writer = new StringWriter();
            PrintWriter print = new PrintWriter(writer, true);
```

Reply:
Well the PrintWriter doesn't override the toString method.

Reply:
Ok got it. So the consequence is a invalid string. Thx.